src/components/Gradebook/index.jsx

```diff
@@ -358,7 +390,8 @@ export default class Gradebook extends React.Component {
             {PageButtons(this.props)}
             <div className="gbook">
               <Table
-                columns={this.props.headings}
+                className={['table-striped']}
```

Comment:
UX/UI request: "Username rows: Starting with the first username row, background color of #f5f5f5 for every other row. More delineations between rows helps clean up the table and makes scanning the middle of the table a little easier"